sjb/config/test_cases/ami_build_origin_int_rhel_base.yml

```diff
@@ -41,8 +41,7 @@ actions:
   - type: "script"
     title: "pre-pull infrastructure images"
     script: |-
```

Comment:
Pull all of them, saves time later

Reply:
Also any steps that failed because the image wasnt there should have the pull image env set to pull